CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.35.0](https://github.com/serverless/serverless/compare/v2.34.0...v2.35.0) (2021-04-09)
+
+### Features
+
+- **AWS Lambda:** Add support for `cacheFrom` to `images` ([#9251](https://github.com/serverless/serverless/issues/9251)) ([341a886](https://github.com/serverless/serverless/commit/341a886874eb8a6c671f576323e75a77cffa1fd2)) ([Nicholas Wehr](https://github.com/wwwehr))
+- **AWS HTTP API:** Adjust schema to allow `serverless/typescript` to generate proper types ([#9275](https://github.com/serverless/serverless/pull/9275)) ([fce73ad](https://github.com/serverless/serverless/commit/fce73ad4df0f776cf950733f9e88c3d390d6fc66)) ([Piotr Grzesik](https://github.com/pgrzesik))
```

Comment:
@pgrzesik Can we also reword it? So it indicates that for property X we allow both direct and array form?

Reply:
We can but that part never changed - we always allowed direct and array forms, but previously we relied on automatic coertion from single value to an array, which wasn't recognized by `serverless/typescript`.

Reply:
> We can but that part never changed - we always allowed direct and array forms, but previously we relied on automatic coertion from single value to an array, which wasn't recognized by serverless/typescript.

hmmm.. I think we 10s+ properties for which this coercion is automatically applied so I'm not sure how picking on one can be a fix on TS (?)

Also I don't think that in this case automatic coercion was really applied, as by AJV internals it's only applied if we deal with array of primitive values, and here this could be array of objects (we allow CF intrinsic functions). So after this change user can use CF intrinsic function directly (which could not work before), and this the improvement I saw with this change.

Reply:
In this case it's also a bug fix that I wasn't aware of - I was under impression that array coercion works not only for primitive values and it was working correctly when e.g. a string was provided as `identitySource`, but now I understand that it wasn't the case when someone would provide an `identitySource` as a CF function, correct?

Reply:
> it wasn't the case when someone would provide an identitySource as a CF function, correct?

Exactly.

With such constructed schama I was not sure how AJV is handling things, but I just checked locally, and if value was string, it was coerced to array, but if an object it was not.

So yes we may consider it as fix, where we allow singular object values which will be automatically resolved into array (and it's how I would put it. Fact that TS had issues in dealing with it is side effect)

Reply:
Thanks a lot for clarification on that as I wasn't aware - I agree that in this case, it's a fix to support all valid notations and fixing TypeScript support is just a side-effect